worlds/dlcquest/data/items.csv

```diff
@@ -28,21 +28,22 @@ id,name,classification,groups
 26,DLC NPC Pack,progression,"DLC,Freemium"
 27,Cut Content Pack,progression,"DLC,Freemium"
 28,Name Change Pack,progression,"DLC,Freemium"
-29,Pickaxe,progression,"Item,Freemium"
 30,Season Pass,progression,"DLC,Freemium"
 31,High Definition Next Gen Pack,filler,"DLC,Freemium"
 32,Increased HP Pack,useful,"DLC,Freemium"
 33,Removed Ads Pack,filler,"DLC,Freemium"
 34,Big Sword Pack,progression,"DLC,Freemium"
 35,Really Big Sword Pack,progression,"DLC,Freemium"
 36,Unfathomable Sword Pack,progression,"DLC,Freemium"
-37,Gun,progression,"Item,DLCQuest"
-38,Sword,progression,"Item,DLCQuest"
-39,Wooden Sword,progression,"Item,Freemium"
```

Comment:
I would personally keep the old items, and mark them as deprecated so they don't get generated, for the sake of backward compatibility.

But even if you do delete them, please don't re-use the same IDs for the new items, as that can cause issues on trackers, text clients, hints, etc.